src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6866,7 +6866,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Compiling requires binding the lambda expression many times. Consider declaring the lambda expression with explicit parameter types, or if the containing method call is generic, consider using explicit type arguments.</value>
   </data>
   <data name="INF_IdentifierConflictWithContextualKeyword" xml:space="preserve">
-    <value>'{0}' is a contextual keyword, with a specific meaning, starting in language version {1}. Use '@{0}' to avoid a breaking change when compiling with language version {1} or later.</value>
+    <value>'{0}' is a contextual keyword, with a specific meaning, in language version {1} or later. Use '@{0}' to avoid a breaking change when compiling with different language versions.</value>
```

Comment:
I wanted wording that made sense regardless of whether the code was compiled with -langversion:12 (or earlier) *or -langversion:13 (or later)*. The previous message said "to avoid a breaking change when compiling with version 13 or later" but that seemed confusing when compiling with 13.

I'm open to suggestions on wording.

Reply:
I don't think the current message adequately describes the issue at all. Here's my workshop:

```suggestion
    <value>'{0}' is a contextual keyword, with a specific meaning in property accessors, in language version {1} or later. Use '@{0}' to avoid potential breaking changes in property accessors when compiling with different language versions.</value>
```

Reply:
'field' is a contextual keyword in property accessors starting in C# 13. Use '@field' instead.

Reply:
How about "to avoid breaking changes ..." rather than "to avoid *potential* breaking changes ..."? (We're only reporting the diagnostic when binding to a different symbol, so the diagnostic is quite likely identifying a breaking change.)

Reply:
I think I like Rikki's suggestion best.